src/WebJobs.Script.WebHost/Diagnostics/SystemLogger.cs

```diff
@@ -82,22 +84,24 @@ public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Except
             // otherwise the ETW event will fail to be persisted (silently)
             string subscriptionId = _subscriptionId ?? string.Empty;
             string appName = _appName ?? string.Empty;
-            string source = _categoryName ?? string.Empty;
+            string source = GetValueFromState(state, ScriptConstants.LogPropertySourceKey) ?? _categoryName;
             string summary = Sanitizer.Sanitize(formattedMessage) ?? string.Empty;
             string innerExceptionType = string.Empty;
             string innerExceptionMessage = string.Empty;
             string functionName = _functionName;
-            string eventName = GetEventName(state);
+            string eventName = GetValueFromState(state, ScriptConstants.LogPropertyEventNameKey);
+            string functionInvocationId = GetValueFromState(state, ScriptConstants.LogPropertyFunctionInvocationIdKey);
+            string hostInstanceId = _hostInstanceId;
+            string activityId = GetValueFromState(state, ScriptConstants.LogPropertyActivityIdKey);
 
             // Populate details from the exception.
             string details = string.Empty;
-            if (string.IsNullOrEmpty(details) && exception != null)
+            if (exception != null)
             {
                 details = Sanitizer.Sanitize(exception.ToFormattedString());
-
                 if (string.IsNullOrEmpty(functionName) && exception is FunctionInvocationException fex)
                 {
-                    functionName = string.IsNullOrEmpty(fex.MethodName) ? string.Empty : fex.MethodName.Replace("Host.Functions.", string.Empty);
+                    functionName = string.IsNullOrEmpty(fex.MethodName) ? string.Empty : fex.MethodName;
```

Comment:
I think we were trying to turn "Host.Functions.FunctionA" into "FunctionA". Is that not needed anymore?

Reply:
This was needed when we were using shortname for the functionName. Webjobs sdk is updated to use logName.

Reply:
The prefixing of the class name in core SDK is a good thing. This is just a special case here we're dealing with, where we've IL genned the wrapper for the customer, so we're trying to remove that name since it doesn't have any logical meaning for them. However in general for users who own the class/method it's good to have the class name as well to help disambiguate.

Reply:
This was unintended change :) Will revert it. @mathewc - functioName here is used to populate our logs and we need it to be consistent so that we can filter on the functionName